scripts/rmc.py

```diff
@@ -117,27 +186,27 @@ def run_visualize(cbmc_filename, cbmc_args, verbose=False, quiet=False, keep_tem
     # 4) cbmc --xml-ui --show-properties ~/rmc/library/rmc/rmc_lib.c temp.goto > property.xml
     # 5) cbmc-viewer --result results.xml --coverage coverage.xml --property property.xml --srcdir . --goto temp.goto --reportdir report
 
-    run_goto_cc(cbmc_filename, temp_goto_filename, verbose, quiet, function=function)
+    run_goto_cc(cbmc_filename, temp_goto_filename, verbose, quiet, function=function, dry_run=dry_run)
     
-    def run_cbmc_local(cbmc_args, output_to):
+    def run_cbmc_local(cbmc_args, output_to, dry_run=False):
         cbmc_cmd = ["cbmc"] + cbmc_args + [temp_goto_filename]
-        return run_cmd(cbmc_cmd, label="cbmc", output_to=output_to, verbose=verbose, quiet=quiet)
+        return run_cmd(cbmc_cmd, label="cbmc", output_to=output_to, verbose=verbose, quiet=quiet, dry_run=dry_run)
 
     cbmc_xml_args = cbmc_args + ["--xml-ui"]
```

Comment:
putting all the property checks into the coverage checks is going to be super expensive. Lets split these out.

Reply:
Added another argument here for coverage checks even though I pass an empty list. The starter kit does not include any by default except for `--malloc-may-fail` and `--malloc-fail-null`, which we did not consider as default flags.